pages/sites/[slug]/[locale]/prd/index.tsx

```diff
@@ -26,6 +26,7 @@ const ProjectListPage: NextPageWithLayout = ({ pageProps, isMobile }) => {
   const { setTenantConfig } = useTenant();
 
   useEffect(() => {
+    localStorage.removeItem('redirectLink');
```

Comment:
Since `redirectLink` is used for the purpose of login, use another name here.

e.g. `backNavigationUrl`

You could also use sessionStorage instead of localStorage (as there is no redirect path).